enumerables.rb

```diff
@@ -0,0 +1,155 @@
+module Enumerable
+    def my_each
+        for i in 0...self.length do
+            yield(self[i])
+        end
+    end
+    
```

Comment:
- [ ] enumerables #my_each returns an Enumerator if no block is given

- [ ]  enumerable  #my_each when a block is given when self is an array returns the array itself. On line 4 convert yield to an array, then return self.